content/en/docs/advanced-tasks/configuring-olm.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Configuring OLM"
+linkTitle: "Configuring OLM"
+weight: 3
+---
+
+# Configuring OLM
+
+The [Operator-lifecycle-Manager (OLM)](https://github.com/operator-framework/operator-lifecycle-manager) controller can be configured through an  [OLMConfig CustomResourceDefinition (CRD)](https://github.com/operator-framework/operator-lifecycle-manager/blob/3f3f00644ca4a50d845db2b97c12afcea56fd5d8/deploy/chart/crds/0000_50_olm_00-olmconfigs.crd.yaml) named `cluster`. This document will outline what configurations OLM currently supports.
```

Comment:
Just had a couple of super quick questions/comments:

- nit: Operator-Lifecycle-Manager
- How do we feel about replacing the OLMConfig CRD link to point to the [API repositories rendered OLMConfig CRD](https://github.com/operator-framework/api/blob/v0.11.0/crds/operators.coreos.com_olmconfigs.yaml) instead of OLM's deploy directory?. I'm not sure whether we'll continue using that deploy directory going forward, and the API repository can act as the source of truth.

Reply:
Ah good point, not sure what I was thinking.

Reply:
Done.